### PR TITLE
Use AdoptOpenJDK JDK 8 and 11

### PR DESCRIPTION
**Pull request by eed3si9n**
Ref https://github.com/scala/scala-dev/issues/587

### CODE REVIEW

**Comment by eed3si9n** [comment]
Travis CI failure is due to an unrelated issue - https://github.com/slick/slick/issues/2051

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
-- docker version
-- sh -v travis/runcontainer.sh oracle db2
-- docker ps
-- sh -v ./travis/extractNonPublicDeps
-jdk:
-  - openjdk8
-  - openjdk11
-  - openjdk12
-notifications:
-  flowdock:
```
Comment (WellingR) [inline .travis.yml]: Has this flowdock stuff been removed on purpose?

**Comment by WellingR** [comment]
Just curious, what is the reason to prefer adoptopenjdk over the openjdk provided by ubuntu xenial?

**Comment by eed3si9n** [comment]
In general Lightbend is standardizing on using AdoptOpenJDK to do testing, but specifically if you want to know why Ubuntu version should be avoided check out 
- https://twitter.com/jroper/status/1062869489073573888
- [Mystery meat OpenJDK builds strike again](https://mail.openjdk.java.net/pipermail/jdk8u-dev/2019-May/009330.html)

File: .travis.yml (hunk 1 above)
Comment (eed3si9n) [reply .travis.yml]: Yes. We no longer use Flowdoc.

**Comment by szeiger** [comment]
Looks good at first glance. Could you rebase this on top of master (with https://github.com/slick/slick/pull/2057 to fix the Travis build)?

**Comment by eed3si9n** [comment]
Travis looks good.

**Comment by d6y** [comment]
Does anyone in this PR thread have an insight into the cause of: https://github.com/slick/slick/issues/2087? (it's `Stop! java 8.0.232.hs-adpt is already installed` as part of `sdk install java`)

**Pull request merged**